lib/remove_person.rb

```diff
@@ -0,0 +1,91 @@
+#!usr/bin/ruby
+
+puts "Enter name of person to remove:"
+name = gets.strip!
+puts "\n###########################\n# Removing #{name}\n###########################"
+
+def short_name(name)
+  name.gsub(" ", "-").downcase
+end
+
+def first_name(name)
+  name.split(" ")[0]
+end
+
+##########################################
+# Remove person from people.yml          #
+##########################################
+
+people_yml_path = "#{Dir.pwd}/data/people.yml"
+tmp_file_path = "#{people_yml_path}.tmp"
+found_person = false
+
+File.open(tmp_file_path, "w") do |output|
+  File.open(people_yml_path, "r") do |input|
+    to_remove_block = false
+    input.each_line do |line|
+      if to_remove_block && line.strip.empty?
```

Comment:
As an alternative, could we treat the file as a datastore and manipulate the data directly, rather than treating it as a syntax file to be parsed.  What I mean is that we use the YAML library to load the contents of people.yml file into a hash, we manipulate the data in the hash, and then use the YAML library to write it back out back into the people.yml file.  That may not preserve the order, but I don't know if the order is important.  If it is important and doing this would change that order then we can make the order more explicit in the data (with an order attribute perhaps?) instead of relying on file order.